packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -533,6 +533,7 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 	public getData( options?: {
 		rootName?: string;
 		trim?: 'empty' | 'none';
+		mergeTagsData?: Record<string, string | ( () => string )>;
```

Comment:
This is not needed here. For such cases, there is `[ key: string ]: unknown;` below.